### ISSUE TITLE
Further procedure with RT

### ISSUE THREAD

**Issue by AlcatrAAz** [robocup-at-work/rulebook#67]
Should the RT be in the rulebook for 2023

Please react this post with the relevant emoji

- [heart] The RT should be this and the upcomming years in the tasks (ATT1 or ATT2 and/or final)

- [party] The RT should be this year a technical challange and next year (and upcomming) back in the tasks (incl note for this)

Feel free to discuss and bring arguments.

**Comment by AlcatrAAz** [comment]
I vote for [heart], the RT should be still in the tasks and not a technical challenge.

I don't see any good reason to remove it for this year and then bring it back next year. The effort to let it in is just changing the numbers in the excel sheet and add a line to the tasks. The effort to remove it completely from the rulebook, write a new technical challenge and revert back everything next year is much higher.

The effort to make the RT easier is for both szenarios the same.

I don't see really the argument that we can test the point system because we changed all test so the point system for the complete competition is new. And normally the points for RT only affect a few teams because only a few teams will try it and even be successfull. Therefore I would increase the points for RT (or even for all special tasks like RT, PPT?) to motivate the teams to try it and invest time to solve the these special tasks.

I would also like to bring the point "consistency" in to play. We rewrote 30 pages last year and brought some new rules. We change the complete structure of the tasks this year. If we now remove the RT too and bring it next year back we will have this and next year again a big change..
